src/gc/gcevents.def

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Personally, the '.def' extension is a bit confusing for me. The def files have very different meaning for me but if everyone else is fine with it then I'm OK too. Not a big deal, just a nit.

Reply:
Agree with @sergiy-k . `.def` extension has a common meaning (Windows linker .def file). I would make this `.h`.

Reply:
OK! Should I also rename this file too? https://github.com/dotnet/coreclr/blob/master/src/gc/gcinterface.dacvars.def

Reply:
Sure. Thank you!